lib/config.py

```diff
@@ -163,6 +37,10 @@ def get_config():
         config_parser.parse()
     return config_parser
 
+# disable argument abbreviation in Python's ArgumentParser class
+class ArgumentParserWithoutAbbrev(argparse.ArgumentParser):
+    def _get_option_tuples(self, option_string):
+        return []
```

Comment:
I believed it was to avoid relying on an option short name (e.g. --mock-a[rgs]) that could be later cause ambiguity with another (e.g. --mock-add), but argparse already takes care of that by raising an exception. Removing.